lib/matplotlib/axes/_axes.py

```diff
@@ -2374,17 +2376,39 @@ def stem(self, *args, **kwargs):
 
         If no *x* values are provided, the default is (0, 1, ..., len(y) - 1)
 
-        Return value is a tuple (*markerline*, *stemlines*,
-        *baseline*).
+        Optional parameters:
```

Comment:
This does not conform with numpy doc specs see https://github.com/matplotlib/matplotlib/pull/6240 for a similar fix. 
If you change it to `Other Parameters` it should work. As it is this section will not be in the web generated version of the docs. 

See https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt for more numpy docs info. 

The output from the docs build should this is in https://travis-ci.org/matplotlib/matplotlib/jobs/118347853#L1036